src/main/resources/institutional-config.properties

```diff
@@ -0,0 +1,144 @@
+app.context.name=kfs
+context.names.rice=cynergy
+production.environment.code=kfs-prod
+logs.directory=/infra/logs
+attachments.directory=/infra/work/attachments
+attachments.pending.directory=/infra/work/attachments/pending
+work.directory=/infra/work
+reports.directory=/infra/work/reports
+staging.directory=/infra/work/staging
+temp.directory=/infra/work/temp
+batch.file.lookup.root.directories=/infra/work/staging;/infra/work/reports
+templates.directory=/infra/work/templates
+attachment.dir.location=/infra/work/attachments
+staging.directory.sys.batchContainer=/infra/work/staging/sys/batchContainer
+data.xml.root.location=/infra/work/staging/workflow
+data.xml.pending.location=/infra/work/staging/workflow/pending
```

Comment:
They were all listed out without placeholders in the template file from puppet I was using as reference, and I also wasn't sure about the interaction between the default properties files, the new institition-config.properties and the command-line property overrides file. That said, you're correct that they would probably benefit from placeholders so let me try it out and see what I can do.